src/XRP/shared/rippled-web-socket-schema.ts

```diff
@@ -0,0 +1,169 @@
+/* Schema for adding type information to Web Socket objects. */
+
+enum WebSocketReadyState {
+  CONNECTING,
+  OPEN,
+  CLOSING,
+  CLOSED,
```

Comment:
is the upper cased-ness of this how ws likes it? typically you case things like `Connecting`